models.py

```diff
@@ -595,18 +595,46 @@ def apply_dither(x: shaping.BT, dither: float):
 	else:
 		return x
 
+
 def entropy(log_probs, lengths = None, dim = 1, eps = 1e-9, sum = True, keepdim = False):
+	""" Calculate the entropy of probabilities, using log of probabilities, and applying the formula:
+			Entropy(p) = -SUM(pj * log pj).
```

Comment:
I would say this is the classic entropy formula:
![image](https://user-images.githubusercontent.com/1041752/105531784-20f59480-5cea-11eb-9dd8-c99ba79ff739.png) (https://en.wikipedia.org/wiki/Entropy_(information_theory))

 it's definitely more "classic" than the formula in L603. So I think the formula in L603 is actually confusing

Reply:
Ok, I agree. Lines 602 & 603 removed.